### DISCUSSION TITLE
Add emoji (like ColorPicker, DatePicker, 🥹 EmojiPicker 🤡)

### DISCUSSION THREAD

**Discussion by alexbakers** [mantinedev/mantine#2425] [Feature requests]
Hello, 

Stickers, memes and **emoji** have become an integral part of our lives 💩

Maybe it's time to add **emoji** to the mantine 🤘🏾

Like [emoji-mart](https://github.com/missive/emoji-mart/)

<img width="374" alt="image" src="https://user-images.githubusercontent.com/98721968/189559444-51cadd0e-9565-43e6-a28f-82b0e1cd71d3.png">

**Comment by rtivital** [comment]
It is in the backlog, it has very low priority though, so it would not be added any time soon
